tensorflow_probability/python/bijectors/ascending.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Ascending bijector."""
+
+import tensorflow.compat.v2 as tf
```

Comment:
What's the intent behind all the backwards-compatibility shims, if tfp 0.11 doesn't even support TF < 2.3? And TF 2 no longer supports python 2.7 either as far as I can see.

Reply:
Just legacy, we only dropped 2.7 support internally at Google several weeks back.

Reply:
So this is going to get cleaned up going forward ?

Reply:
i think this compat shim is still needed for lingering TF1 users internal to google. we probably could clean it up on our code export rewrites.

Reply:
(That said, let's leave it here for now)